### PR TITLE
Update E2E test command

### PR DESCRIPTION
**Pull request by hhpatel14**
-  Additional cluster is not required for CI unit test. 
-  Update temporary Unit to E2E.

### CODE REVIEW

File: ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml [hunk 1]
```diff
@@ -20,12 +20,22 @@ images:
       as:
       - quay.io/app-sre/boilerplate:image-v2.1.0
   to: oadp-operator
+- dockerfile_path: build/ci-Dockerfile
+  from: ocp_builder_rhel-8-golang-1.16-openshift-4.9
+  to: test-oadp-operator
```
Comment (stevekuznetsov) [inline ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml:25]: This entry builds an image called `test-oadp-operator`.

File: ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml [hunk 2]
```diff
@@ -20,12 +20,22 @@ images:
...
 operator:
   bundles:
   - dockerfile_path: build/Dockerfile.bundle
   substitutions:
   - pullspec: quay.io/konveyor/oadp-operator:latest
-    with: oadp-operator
+    with: test-oadp-operator
+promotion:
+  additional_images:
+    test: test-oadp-operator
```
Comment (stevekuznetsov) [inline ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml:34]: This entry promotes the `test-oadp-operator` to a central registry when we promote - after merges happen to the repo.

File: ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml [hunk 3]
```diff
@@ -39,6 +49,20 @@ resources:
...
+- as: operator-unit-test
+  steps:
+    test:
+    - as: unit
+      commands: make test
+      credentials:
+      - mount_path: /var/run/oadp-credentials
+        name: oadp-credentials
+        namespace: test-credentials
+      from: test
```
Comment (stevekuznetsov) [inline ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml:61]: This tries to use an unknown name for the image - change this to `test-oadp-operator`.

File: ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml (hunk 2 above)
Comment (hhpatel14) [reply ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml:34]: so, pipeline: test-oadp-operator to openshift/test-oadp-operator:test ?

**Pull request closed without merging**